src/main/java/com/google/songgame/servlets/GameServlet.java

```diff
@@ -1,43 +1,50 @@
 package com.google.songgame.servlets;
 
-import com.google.gson.Gson;
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.YouTubeRequestInitializer;
+import com.google.api.services.youtube.model.PlaylistItemListResponse;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Random;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import com.wrapper.spotify.SpotifyApi;
-import com.wrapper.spotify.exceptions.SpotifyWebApiException;
-import com.wrapper.spotify.model_objects.specification.Playlist;
-import com.wrapper.spotify.model_objects.specification.PlaylistTrack;
-import com.wrapper.spotify.requests.data.playlists.GetPlaylistRequest;
-import com.wrapper.spotify.requests.data.playlists.GetPlaylistsItemsRequest;
-import com.wrapper.spotify.model_objects.credentials.ClientCredentials;
-import com.wrapper.spotify.requests.authorization.client_credentials.ClientCredentialsRequest;
-import org.apache.hc.core5.http.ParseException;
-import com.google.gson.Gson; 
-import java.util.ArrayList;
-
 @WebServlet("/game")
 public final class GameServlet extends HttpServlet {
+
+  private static final String DEVELOPER_KEY = "AIzaSyBZw4Z25Lect7ux9z960RCM7YORcYo6slc";
+  private static final String CLIENT_SECRETS = "/client_secret.json";
+  private static final Collection<String> SCOPES =
+      Arrays.asList("https://www.googleapis.com/auth/youtube.readonly");
+
+  private static final String APPLICATION_NAME = "Song Guessing Game";
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
```

Comment:
Is it possible to use GsonFactory.getDefaultInstance() here instead?
https://googleapis.dev/java/google-http-client/latest/com/google/api/client/json/gson/GsonFactory.html#getDefaultInstance--

Since we're already using the Gson library in this servlet and in CharServlet, I don't think we should introduce another Json library unless we have to.